bpa-demo/src/styles.scss

```diff
@@ -5,7 +5,8 @@
 html, body {
    height: 100%;
    width: 100%;
-   padding: 5px;
+   //padding: 5px;
+   margin: 0px !important;
```

Comment:
why !important. It should only be used to override third party styles(If not possible through nesting)